conf/base.config

```diff
@@ -11,7 +11,7 @@
 
 process {
 
-  container = params.container
+  container = "${params.containerPath}/rnafusion-${params.container_version}.img"
```

Comment:
We want to keep `base.config` as it is. You have to specify your own profile where you define the path to the downloaded singularity images. Check the [munin-singularity.config](https://github.com/nf-core/rnafusion/blob/master/conf/munin-singularity.config).